Added CombinedLR version

wavtype opened the pull request:
Under testing (prototypes ordered)

wavtype committed:
added LR version

wavtype committed:
LR footprints

wavtype committed:
Create Gerber.zip

wavtype committed:
fixed LR

working on mouseclick LR boards.

wavtype committed:
fixed LR board mouse switch footprint

wavtype committed:
removed gerber

wavtype committed:
flipped trigger boards

CarVac commented:
https://media.discordapp.net/attachments/931041368141332500/981264817606234172/2022-05-31_14_35_45-Mailspring_Order_W202206010055338_is_suspended_production_JLCPCB.jpg

Apparently JLC wants a more robust tab for the c-stick board when doing SMT assembly. That might affect our ability to have really small ones here.

wavtype committed:
Silk & Footprint Rework

Made minor edits to the footprints that were causing keepout area errors and silkscreen clearance errors to make it easier for newer designers to make sense of DRC (only error is GND plane of C-stick to GND plane of main PCB). removed labels from mounting holes and other non essential items (stickboxes, trigger sliders, teensy, some header pins) and labeled all buttons and important header pins. also moved some of the component labels so they lined up a bit better with their respective components.

CarVac commented:
I tried opening the pcb file from Combined but it's broken.

![Screenshot_20220531_235806](https://user-images.githubusercontent.com/4342361/171324683-a41a289d-3524-4c0e-92fd-8d4cbe76aca3.png)

wavtype committed:
silk and footprint rework (combined)

wavtype committed:
swapped fuse, fixed LR

wavtype commented:
> I tried opening the pcb file from Combined but it's broken.
> 
> ![Screenshot_20220531_235806](https://user-images.githubusercontent.com/4342361/171324683-a41a289d-3524-4c0e-92fd-8d4cbe76aca3.png)

yeah had a bad string of events after accidentally doing the work in you dpad branch (oops) should be fixed now

wavtype committed:
wide mousebites

wavtype committed:
Delete Gerber.zip

wavtype committed:
fixed footprints

wavtype committed:
applied footprint changes to all pcbs

forgot to change cstick and motherboard to reflect the new hall sensor footprint, fixed now.

wavtype committed:
fixed data line connection.
